### ISSUE TITLE
Install button does not work

### ISSUE THREAD

**Issue by JPSchellenberg** [Lumieducation/Lumi#1756]
**Describe the bug**
The install button from the settings -> updates menu does not work.

**Environment (please complete the following information):**
v0.8.1

**To Reproduce**
1. Go to Settings
2. Go to Updates
3. Click "Install and restart" button

**Expected behavior**
Should download, install and restart the application.

**Screenshots**
![Screenshot 2021-08-06 at 08 36 47](https://user-images.githubusercontent.com/9697985/128467039-4980029a-879a-47a3-8651-f5d38084a61d.png)

**Additional context**

**Comment by sr258** [comment]
Does its not work in general or does it simply not work because the version is already current?

**Comment by JPSchellenberg** [comment]
No, I had version 0.8.0-beta.5 and it did not work. We should also hide it if it is the newest version.
